bootcamp/tutorials/quickstart/multimodal_rag_with_milvus.ipynb

```diff
@@ -0,0 +1,884 @@
+{
```

Comment:
<p>Data should be prepared in advance. Split the section into 2 sections: "Prepare Data" and "Create Collection".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1398/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>